docs/sources/release-notes/v2.2.md

```diff
@@ -0,0 +1,45 @@
+weight: 300
+---
+
+# Grafana Mimir version 2.2 release notes
+
+Grafana Labs is excited to announce version 2.2 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+Below we highlight the top features, enhancements and bugfixes in this release, as well as relevant callouts for those upgrading from Grafana Mimir 2.1.
+The complete list of changes is recorded in the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for ingesting out-of-order samples**: Grafana Mimir introduces new experimental support for ingesting out-of-order samples.
+  This support is configurable, with users able to set how far out-of-order Mimir will accept samples on a per-tenant basis.
+  **TODO: Link to documentation after https://github.com/grafana/mimir/pull/2183 is merged.**
+
+- **Backfill API**: Grafana Mimir 2.2 introduces new API for backfill. TODO: expand once mimirtool changes are in.
+
+- **Error messages**: We have made the error messages reported by Mimir more human readable and included error codes that can be easily searched for.
+  These error message improvements will continue in future releases.
+  You can find description of errors in [Grafana Mimir Runbooks / Codified errors]({{< relref "../operators-guide/mimir-runbooks/_index.md#codified-errors" >}}).
+
+- **Configurable prefix for object storage**: Mimir can now store block data, rules, and alerts in one bucket, each under its own user-defined prefix, rather than requiring one bucket for each. Storage prefix can be configured using `-<storage>.storage-prefix` option for corresponding storage: `ruler-storage`, `alertmanager-storage` or `blocks-storage`.
+
+## Upgrade considerations
+
+We have updated default values and some parameters in Grafana Mimir 2.2 to give users better out-of-the-box experience:
+
+- Message size limits for gRPC messages exchanged between internal Mimir components has been increased to 100 MiB from previous 4 MiB. This helps to avoid internal server errors when pushing or querying large data.
+
+- The `-blocks-storage.bucket-store.ignore-blocks-within` parameter has been changed from `0` to `10h`. The default value of `-querier.query-store-after` has been changed from `0` to `12h`. Both changes improve query-performance for most-recent data by querying only the ingesters, rather than object storage.
+
+The option `-querier.shuffle-sharding-ingesters-lookback-period` has been deprecated. Anyone who previously changed this option from its default of `0s` should instead set `-querier.shuffle-sharding-ingesters-enabled` to `true` and specify the lookback period by setting the `-querier.query-ingesters-within` option.
+
+- The `-memberlist.abort-if-join-fails` parameter now defaults to false. This means that when using memberlist and a component fails to join the memberlist cluster, it no longer aborts startup by default.
```

Comment:
Clarify what "it" is in the phrase,  "it no longer aborts startup by default."

Grammatically, "it" can mean the component, the parameter, or the cluster, or even possibly some assumed knowledge that is implicit.

Reply:
"it" here refers to Mimir component. WDYT about this instead?

```suggestion
- The `-memberlist.abort-if-join-fails` parameter now defaults to false. This means that when using memberlist and a Mimir fails to join the memberlist cluster, Mimir no longer aborts startup by default.
```

Reply:
I've used this for now:

> - The `-memberlist.abort-if-join-fails` parameter now defaults to false. When Mimir is using memberlist as a backend store for hash ring, and it fails to join the memberlist cluster, Mimir no longer aborts startup by default.